docs/requirements.txt

```diff
@@ -1,4 +1,4 @@
-mkdocs-material==9.5.3
+mkdocs-material==9.5.6
```

Comment:
This shouldn't be a problem, I was planning to merge the dependabot update anyways.